.github/workflows/linux.yml

```diff
@@ -50,9 +50,17 @@ jobs:
         cd
         rm -rf astyle*
         astyle --version
-    - name: indent
+    - name: make indent
       run: |
         ./contrib/utilities/indent
+        git diff > changes-astyle.diff
+    - name: archive indent results
+      uses: actions/upload-artifact@v2
+      with:
+        name: changes-astyle.diff
+        path: changes-astyle.diff
+    - name: check indentation
+      run: |
         git diff --exit-code
```

Comment:
You could presumably make this a bit cheaper by just saying "cat changes-astyle.diff". Or do you need the return code of `git diff` here?

Reply:
I need the return code to let the test crash if indentation changes are necessary, I could probably check if `changes-astyle.diff` contains anything, but I do not think it will save more than a few seconds.